src/index.ts

```diff
@@ -144,6 +149,12 @@ export interface User {
   postalCode?: string
   addressLine1?: string
   addressLine2?: string
+
+  // typeCode is the identifier representing the data subject type specified by the user
+  typeCode?: string
+
+  // typeDescription is additional information provided by the user describing their relation to the business
+  typeDescription?: string
```

Comment:
how is this different to the `description` above?

Reply:
It is a separate text box directly under the data subject dropdown that specifically is for information about how the data subject related to the business. Don't think there are screens here yet but asked in figma